### DISCUSSION TITLE
Fresh 2.4 installation from ISO : kernel panic - not syncing: attempted to kill init exitcode 0x00007f00

### DISCUSSION THREAD

**Discussion by sirtcp** [Security-Onion-Solutions/securityonion#12496] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

25GB

### Storage for /

100

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi, 

I've been using 2.3 for the last few months on Proxmox VE, and it has been working perfectly. I love the product. However, I want to upgrade to 2.4. The problem is, upon boot, I encounter a kernel panic error. I've tried both versions 2.4.50 and 2.4.40, and I've verified the signatures as well. However, during the initial boot after choosing Grub, none of the options I've tried on both versions seem to move forward. Instead, they end up with the following error: "kernel panic - not syncing: attempted to kill init exitcode 0x00007f00." Unfortunately, I can't see the complete error because the Proxmox boot resolution is very small.

Regards,
Yousuf

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Is your VM CPU set to `host`? https://docs.securityonion.net/en/2.4/proxmox.html#cpu

**Comment by sirtcp** [comment]
Thank you, I missed this part. now it's working
Thank you for the update.
Regards,
Yousuf
